pkg/controllers/cluster/cluster_controller.go

```diff
@@ -238,9 +254,46 @@ func (c *Controller) removeCluster(cluster *clusterv1alpha1.Cluster) (controller
 	// delete the health data from the map explicitly after we removing the cluster.
 	c.clusterHealthMap.delete(cluster.Name)
 
+	// check if target cluster is removed from all bindings.
+	if c.EnableTaintManager {
+		if done, err := c.isTargetClusterRemoved(ctx, cluster); err != nil {
+			klog.ErrorS(err, "Failed to check whether target cluster is removed from bindings", "cluster", cluster.Name)
```

Comment:
https://github.com/karmada-io/karmada/pull/2373/files/d468101a705ce83fd6ffc3022fba2a8199b1c7ca#r945591288

I mean here we checked the `c.EnableTaintManager`. right?

Reply:
Oh, I see.